### PR TITLE
Wrong description for the 'Pod Anti-affinity' and 'Pod Affinity with no Matching Labels'

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1756248

### CODE REVIEW

**Comment by mburke5678** [comment]
@zhouying7780  
Thank you for pointing this out. Please take a look to make sure I addressed your concerns.

**Comment by mburke5678** [comment]
Change for 3.x: https://github.com/openshift/openshift-docs/pull/17022

File: modules/nodes-scheduler-pod-affinity-example.adoc [hunk 1]
```diff
@@ -103,12 +103,7 @@ spec:
     image: docker.io/ocpqe/hello-pod
 ----
 
-* The pod *pod-s2* is not scheduled unless there is a node with a pod that has the `security:s2` label. If there is no other pod with that label, the new pod remains in a pending state:
-+
-----
-NAME      READY     STATUS    RESTARTS   AGE       IP        NODE
-pod-s2    0/1       Pending   0          32s       <none>
-----
+* The pod *pod-s2* cannot be scheduled on the same node as `pod-s1`.
```
Comment (zhouying7780) [inline modules/nodes-scheduler-pod-affinity-example.adoc]: In the example yaml file  the first pod name is `s1`, so here should be `s1`.

**Comment by zhouying7780** [comment]
Other LGTM .

File: modules/nodes-scheduler-pod-affinity-example.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/nodes-scheduler-pod-affinity-example.adoc]: @zhouying7780 Thank you for catching that. I changed the first pod name to pod-s1.

**Pull request merged**